recipe/meta.yaml

```diff
@@ -12,16 +12,15 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1002
-  skip: true  # [not py2k]
+  number: 1003
+  noarch: python
```

Comment:
Switched the package to `noarch: python` so we only need to worry about one CI build for creating the package. Though please correct me if the package contains compiled code or does something else special with Python versions at build time.

Reply:
There is no compiled code indeed.